evm/src/main/java/org/hyperledger/besu/evm/operation/SelfDestructOperation.java

```diff
@@ -60,8 +60,9 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
     final boolean beneficiaryIsWarm =
         frame.warmUpAddress(beneficiaryAddress) || gasCalculator().isPrecompile(beneficiaryAddress);
 
-    final Address originatorAddress = frame.getRecipientAddress();
-    final Wei originatorBalance = frame.getWorldUpdater().get(originatorAddress).getBalance();
+    final Address originatorAddresss = frame.getRecipientAddress();
+    final MutableAccount originatorAccount = frame.getWorldUpdater().getAccount(originatorAddresss);
```

Comment:
spelling nitpick (sss).  also originatorAddress is unused apart from getAccount.

Reply:
Had it fixed in another stash. ;)